src/debug.c

```diff
@@ -713,7 +713,10 @@ NULL
         if (getPositiveLongFromObjectOrReply(c, c->argv[2], &keys, NULL) != C_OK)
             return;
 
-        dictExpand(c->db->dict,keys);
+        if (dictTryExpand(c->db->dict, keys) != DICT_OK) {
```

Comment:
I see `dictExpands` calls `dictExpand(c->db->dict,keys)` internally and it includes other errors besides the malloc error. Shall we process the error such as the below?

https://github.com/redis/redis/blob/6bf9b144ef722fb301c065125a6bf6cd385841bf/src/dict.c#L240-L241

Reply:
thanks for the looking, actually we don't need to check the other errors
dictTryExpand it will only return DICT_ERR if malloc fails, and actually in this case we really don't need to care about other errors, they are considered DICT_OK by dictTryExpand

Reply:
Ok, I get it.